Source/AliveLibAE/Abe.cpp

```diff
@@ -3175,7 +3175,11 @@ void Abe::State_0_Idle_44EEB0()
 
             case TlvTypes::GrenadeMachine_59:
             {
-                BoomMachine* pMachineButton = static_cast<BoomMachine*>(FindObjectOfType_425180(Types::eGrenadeMachine_66, field_B8_xpos, field_BC_ypos - (field_CC_sprite_scale * FP_FromInteger(25))));
+                BoomMachine* pMachineButton = static_cast<BoomMachine*>(FindObjectOfType_425180(
```

Comment:
you used auto in the others but not here ;)

Reply:
maybe because I didn't do this line, and just placed endl in it :/